var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -72,6 +72,15 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
     version("master", branch="master")
     version("develop", branch="develop")
 
+    depends_on("c", type="build")  # generated
+    depends_on("cxx", type="build")  # generated
+    depends_on("fortran", type="build")  # generated
```

Comment:
Please remove the comment if you've confirmed that the package does depend on these compilers.